### ISSUE TITLE
Create a BOM for required parts

### ISSUE THREAD

**Issue by hadesto** [pr-holonet/pr-holonet#7]
create a bill of materials of the components needed to build a pr-holonet node

**Comment by ewanmellor** [comment]
We currently have:

* RPi
* RockBLOCK
* 1 RGB shared-cathode LED
* 1 single-color LED
* 4 resistors, sized to suit the LEDs
* 1 female-female jumper cable between the RockBLOCK and the Pi
* further cabling between the LEDs and the RPi GPIO pins (not done)
* A case (not done)

Power is supplied by micro-USB to the Pi.  I presume that you would simply expose this port outside the case.  However, if you wanted to include a battery, that would obviously need to be thought about carefully.

I presume that you're going to want some kind of LED package, with the two LEDs, resistors, and pins or a port for cabling to the Pi.

I don't know whether the jumper cable would be sufficient for the RockBLOCK -> Pi connection or you would want to make a single cable so that it was easier to assemble and less likely to come apart.

**Comment by hadesto** [comment]
I've thought about the battery. Not sure yet though. If we used a LiPo we'd have to include circuitry that could keep it from overcharging the battery, as well to keep it from fully draining it. Not to mention temperature constraints as well as regulations related to the battery itself.

In my mind, I think we should keep this as simple as possible so that anyone could build it with relatively easily accessible parts. So for now I say we stick to the USB port and power it off externally. In any case, I would argue cellphone battery chargers are pretty commonplace these days. That coupled with a solar cell should make it a viable off-grid power option.

**Comment by ewanmellor** [comment]
Yes, I agree that using only external power / battery is a better idea.
